transform/mattermost-analytics/models/staging/mm_calls_test_go/stg_mm_calls_test_go__tracks.sql

```diff
@@ -0,0 +1,22 @@
+WITH tracks AS (
+    SELECT
+        {{ dbt_utils.star(ref('base_mm_calls_test_go__tracks')) }}
+    FROM
+        {{ ref ('base_mm_calls_test_go__tracks') }}
+)
+SELECT
+        participants as participants,
+        server_version as server_version,
+        call_id as call_id,
+        plugin_build as plugin_build,
+        event as event,
+        user_id as server_id,
+        received_at as received_at,
+        plugin_version as plugin_version,
+        duration as duration,
```

Comment:
Please see the comment in base table regarding the list of columns.

Reply:
Removed these columns, as they aren't required at this moment. Will add them in a separate table if requested.